tests/DataValues/DecimalValueTest.php

```diff
@@ -42,6 +42,7 @@ public function validConstructorArgumentsProvider() {
 		$argLists[] = array( '+0.0' );
 		$argLists[] = array( '+0.000' );
 		$argLists[] = array( '+1.0' . str_repeat( ' ', 124 ) );
+		$argLists[] = array( '4.2' );
```

Comment:
`' 4.2'` should also be valid no?

Also need to verify the value gets interpreted correctly. If I'm not mistaken, you could change the production code to add a minus instead of a plus now, and the tests would still pass.